src/bijectors/simplex.jl

```diff
@@ -37,10 +49,22 @@ function (b::SimplexBijector{proj})(x::AbstractVector{T}) where {T, proj}
 end
 
 # Vectorised implementation of the above.
-function (b::SimplexBijector{proj})(X::AbstractMatrix{T}) where {T<:Real, proj}
-    Y, K, N = similar(X), size(X, 1), size(X, 2)
+function (b::SimplexBijector)(X::AbstractMatrix)
+    _simplex_bijector(X, b)
+end
+function (b::SimplexBijector)(
+    Y::AbstractMatrix,
+    X::AbstractMatrix,
+)
+    _simplex_bijector!(Y, X, b)
```

Comment:
Seems like if some bijectors suddenly start taking multiple arguments, things can be a bit confusing. Granted, I guess we could establish the convention that if a `Bijector` takes two arguments, it's _always_ inplace?

This also kind of brings up the discussion we had waaay back about whether to use `(b::Bijector)(x)` or `transform(b, x)` (thus also allowing `transform!(b, x, y)`).

Reply:
Hmm just curious what bijector takes 2 arguments? Can we just make the convention that it always takes 1, maybe a tuple if we really have to?